packages/uploadthing/src/remix.ts

```diff
@@ -0,0 +1,35 @@
+import type { ActionFunctionArgs } from "@remix-run/server-runtime";
+import * as Effect from "effect/Effect";
+
+import type { Json } from "@uploadthing/shared";
+
+import { makeAdapterHandler } from "./internal/handler";
+import type { FileRouter, RouteHandlerOptions } from "./internal/types";
+import type { CreateBuilderOptions } from "./internal/upload-builder";
+import { createBuilder } from "./internal/upload-builder";
+
+export type { FileRouter };
+export { UTFiles } from "./internal/types";
+
+type MiddlewareArgs = {
+  req: Request;
```

Comment:
should this also be undefined? push the full action args on `event` instead?

they have `event.request` if they need to but im getting the vibes that they use the full args more 🤔